protocol/src/main/java/org/opensearch/sql/protocol/response/format/NoQueryResponseFormatter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.protocol.response.format;
+
+import lombok.Getter;
+import org.opensearch.sql.protocol.response.QueryResult;
+
+public class NoQueryResponseFormatter extends JsonResponseFormatter<QueryResult> {
```

Comment:
TODO: add javadoc

Reply:
Would `EmptyResponseFormatter` be better?

Reply:
`CommandResponseFormatter` to book-end `CommandPlan` abstract plan?